pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
```

Comment:
I wouldn't choose setuptools as the build backend as it has to support many legacy workflows that prevent innovations in other tools. I would propose `hatchling.build` as the official tutorial does https://packaging.python.org/en/latest/tutorials/packaging-projects/#creating-pyproject-toml (although implicitly as listing it as the first choice)